ci/jenkins/test.sh

```diff
@@ -323,7 +324,11 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
```

Comment:
How about adding `SNAPSHOT_NAME=$2` ? For the Docker worker nodes, we still need to use the 'win-initial' snapshot because this new feature is only supported by the containerd runtime.
https://github.com/antrea-io/antrea/blob/f32f7963842617487c5859b0cb28e789af7132f0/ci/jenkins/test.sh#L413

Reply:
done, thanks!